packages/dds/tree/src/shared-tree/treeAlpha.ts

```diff
@@ -357,11 +361,11 @@ export interface TreeAlpha {
 	/**
 	 * Gets the child of the given node with the given property key if a child exists under that key.
 	 *
-	 * @remarks Unknown optional fields of Object nodes cannot be accessed using this method.
+	 * @remarks {@link SchemaFactoryObjectOptions.allowUnknownOptionalFields | Unknown optional fields}  of Object nodes cannot be accessed using this method.
```

Comment:
nit: Maybe "will return undefined" rather than "cannot be accessed" - the latter could be interpreted as an illegal operation (e.g. that throws an error).